src/mod/server.mod/servmsg.c

```diff
@@ -493,7 +493,7 @@ static int detect_flood(char *floodnick, char *floodhost, char *from, int which)
   if (!strcasecmp(floodhost, botuserhost))
     return 0;
 
-  u = get_user_by_host(from);
```

Comment:
can't we get an account into this function, same as the other flood function?

Reply:
Not without making big changes to the larger codebase